prow/plugins/trigger/generic-comment_test.go

```diff
@@ -228,6 +228,28 @@ func TestHandleGenericComment(t *testing.T) {
 			ShouldBuild:   true,
 			StartsExactly: "pull-jib",
 		},
+		{
+			name:   "test of silly regex job",
+			Author: "trusted-member",
+			Body:   "Nice weather outside, right?",
+			State:  "open",
+			IsPR:   true,
+			Presubmits: map[string][]config.Presubmit{
+				"org/repo": {
+					{
+						JobBase: config.JobBase{
+							Name: "jab",
```

Comment:
Nit: https://github.com/kubernetes/test-infra/pull/11168#issuecomment-461643114
🤔

Reply:
The shade is so real ;)